spec/datadog/appsec/component_spec.rb

```diff
@@ -38,9 +39,10 @@
 
         it 'returns a Datadog::AppSec::Component instance with a nil processor and does not warn' do
```

Comment:
I think it's not matching anymore?

Reply:
thanks! fixed [6b70f5e](https://github.com/DataDog/dd-trace-rb/pull/4578/commits/6b70f5e6489ba3b16000e79282b22047d4dd4519)